docs/ENTITY_RELATIONSHIP_DIAGRAM.md

```diff
@@ -90,9 +104,13 @@ erDiagram
   Proposal ||--|{ ProposalVersion : has
   ProposalVersion ||--|{ ProposalFieldValue : contains
   ProposalFieldValue }o--|| ApplicationFormField : populates
-  Applicant ||--o{ ExternalFieldValue : "is described by"
+  Organization ||--o{ ExternalFieldValue : "is described by"
   ExternalFieldValue }o--|| BaseField : "contains potential defaults for"
-  ExternalSource ||--o{ ExternalFieldValue : "populates"
+  Source }|--o| Funder : "represents"
+  Source }|--o| Organization : "represents"
+  Source }|--o| DataProvider : "represents"
+  ProposalVersion }o--|| Source : "comes from"
```

Comment:
It might be good to (also?) relate `Source` to the finer-grained `ProposalFieldValue` (like we do to `ExternalFieldValue`). There may (eventually) be times when a `ProposalFieldValue` gets copied from another proposal or from an `ExternalFieldValue` and if it really is copied then preserving the original actual `Source` in the copy would be nice. This also might be one of those "we can do it when the situation actually happens" things.

Reply:
Yes!  FWIW this is still WIP (though I appreciate the review so far) -- that's the next feature to add.

Reply:
Agree it can be deferred to when actually needed.